status/statusgo_backend_new/stickers.nim

```diff
@@ -0,0 +1,74 @@
+import json, stint, web3/[ethtypes, conversions], strutils, tables, sequtils
+import
+  ../types/[setting],
+  ../transactions,
+  ../utils as status_utils,
+  ../statusgo_backend/wallet,
+  ../eth/contracts as eth_contracts,
+  ../eth/stickers as eth_stickers,
+  ../eth/methods as eth_methods,
+  ../statusgo_backend/settings as status_settings,
+  ../types/[sticker, transaction, rpc_response, network_type, network]
+
+proc buildTransaction(
+    packId: Uint256,
+    address: Address,
+    price: Uint256,
+    approveAndCall: var ApproveAndCall[100],
+    sntContract: var Erc20Contract,
+    gas = "",
+    gasPrice = "",
+    isEIP1559Enabled = false,
+    maxPriorityFeePerGas = "",
+    maxFeePerGas = ""
+    ): TransactionData =
+  let network = status_settings.getCurrentNetwork().toNetwork()
+  sntContract = eth_contracts.findErc20Contract(network.chainId, network.sntSymbol())
+  let
+    stickerMktContract = eth_contracts.findContract(network.chainId, "sticker-market")
+    buyToken = BuyToken(packId: packId, address: address, price: price)
+    buyTxAbiEncoded = stickerMktContract.methods["buyToken"].encodeAbi(buyToken)
+  approveAndCall = ApproveAndCall[100](to: stickerMktContract.address, value: price, data: DynamicBytes[100].fromHex(buyTxAbiEncoded))
+  transactions.buildTokenTransaction(address, sntContract.address, gas, gasPrice, isEIP1559Enabled, maxPriorityFeePerGas, maxFeePerGas)
+
+proc getInstalledStickerPacks*(): JsonNode =
```

Comment:
@jrainville it's not part of the lib to this parsing, it's only here as a bridge for communication with `status-go`.
All methods from the `status-lib` should have the same return type `RpcResponse[JsonNode] {.raises: [Exception].}` and that will be handled in appropriate service on the `status-desktop` side.